src/main/java/seedu/address/logic/commands/CheckoutCommand.java

```diff
@@ -30,10 +34,15 @@ public CheckoutCommand(RoomNumber roomNumber) {
     @Override
     public CommandResult execute(Model model, CommandHistory history) throws CommandException {
         requireNonNull(model);
-
-        // roomNumber is guaranteed to be a valid room number.
+        // roomNumber is guaranteed to be a valid room number after parsing.
+        if (!model.hasActiveBooking(roomNumber)) {
+            throw new CommandException(String.format(MESSAGE_NO_ACTIVE_BOOKING_ROOM_CHECKOUT, roomNumber));
```

Comment:
Will cancelling of reservations be done via the checkout command? 

What's the value add for adding this?

Reply:
No. Checkout strictly checks out an active booking that has been checked in.

Cancelling of reservations must be done via another command.